### DISCUSSION TITLE
[Support]: Can't see snapshots in the UI after upgrade

### DISCUSSION THREAD

**Discussion by lee-costa** [blakeblackshear/frigate#12931] [General Support] [answered]
### Describe the problem you are having

It was useful being able to see snapshot under the old UI in order to debug exactly how it was triggered but I can't seem to find a way to review it now. Do I need to upgrade to +?

### Version

0.14

### Frigate config file

```yaml
//
```

### Relevant log output

```shell
//
```

### FFprobe output from your camera

```shell
//
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Amcrest

**Comment by NickM-27** [comment, marked as answer]
The frigate+ view (which just requires a free account) can be used for this. There are plans for a separate view in the future.

**Comment by hawkeye217** [comment]
Feel free to leave a thumbs up on this feature request as well: https://github.com/blakeblackshear/frigate/issues/11399

**Comment by hawkeye217** [comment]
Also, see the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 8 for more on why snapshots were deprecated in the new UI for 0.14.
